mathics/builtin/fileformats/htmlformat.py

```diff
@@ -47,37 +48,39 @@ def children():
 
     def attributes():
         for name, value in node.attrib.items():
-            yield Expression("Rule", from_python(name), from_python(value))
+            yield to_expression(SymbolRule, from_python(name), from_python(value))
 
     return [
-        Expression(
+        to_expression(
             "XMLElement",
             String(node.tag),
-            Expression("List", *list(attributes())),
-            Expression("List", *list(children())),
+            to_expression("List", *list(attributes())),
+            to_expression("List", *list(children())),
         )
     ]
 
 
 def xml_object(tree):
     declaration = [
         Expression(
-            Expression("XMLObject", String("Declaration")),
-            Expression(
-                "Rule", String("Version"), String(tree.docinfo.xml_version or "1.0")
+            to_expression("XMLObject", String("Declaration")),
+            to_expression(
+                SymbolRule, String("Version"), String(tree.docinfo.xml_version or "1.0")
             ),
-            Expression(
-                "Rule",
+            to_expression(
+                SymbolRule,
                 String("Standalone"),
                 String("yes") if tree.docinfo.standalone else String("no"),
             ),
-            Expression("Rule", String("Encoding"), String(tree.docinfo.encoding)),
+            to_expression(
+                SymbolRule, String("Encoding"), String(tree.docinfo.encoding)
+            ),
         )
     ]
 
     return Expression(
-        Expression("XMLObject", String("Document")),
-        Expression("List", *declaration),
+        to_expression("XMLObject", String("Document")),
+        to_expression("List", *declaration),
```

Comment:
I don't think it is _that_ much work use `to_mathics_list()` here. 

But if you want to keep going and do the others, I'll just make the simple changes here in this PR.